references/detection/transforms.py

```diff
@@ -53,6 +53,22 @@ def forward(
         return image, target
 
 
+class ToDtype(nn.Module):
+    def __init__(self, dtype: torch.dtype, scale: bool = False) -> None:
+        super().__init__()
+        self.dtype = dtype
+        self.scale = scale
+
+    def forward(
+        self, image: Tensor, target: Optional[Dict[str, Tensor]] = None
+    ) -> Tuple[Tensor, Optional[Dict[str, Tensor]]]:
+        if not self.scale:
+            return image.to(dtype=self.dtype), target
+        image = F.convert_image_dtype(image, self.dtype)
+        return image, target
+
+
+# Keep ConvertImageDtype for BC
```

Comment:
We don't really need to provide BC in those reference transforms. We've been changing a bunch of things fairly often. I think it'd be best to just rename `ConvertImageDtype` below into `ToDtype` and avoid duplication

Reply:
yeah we don't have a way for it to ignore specific directories. But it's safe to ignore for everything in the references